cmd/zoekt-archive-index/e2e_test.go

```diff
@@ -90,7 +92,10 @@ func testIndexIncrementally(t *testing.T, format string) {
 	if err != nil {
 		t.Fatalf("TempDir: %v", err)
 	}
-	defer os.RemoveAll(indexdir)
+	defer func() {
+		advanceFS()
```

Comment:
tests don't pass without waiting before removing the directory. This is inspired by `watcher_test.go`.

Reply:
Not relevant anymore